src/language/language.rs

```diff
@@ -562,6 +562,7 @@ impl Display for PadType {
 pub enum MyAnalysisData {
     Literal(ndarray::ArrayD<f64>),
     Legacy(MyAnalysisDataLegacyData),
```

Comment:
```suggestion
```
We should remove this! If everything is good here, then removing this shouldn't throw any errors.